cookbooks/bcpc-hadoop/recipes/hbase_master.rb

```diff
@@ -65,3 +66,20 @@
   subscribes :restart, "user_ulimit[hbase]", :delayed
   subscribes :restart, "template[/etc/hadoop/conf/core-site.xml]", :delayed
 end
+
+if node["bcpc"]["hadoop"]["phoenix"]["tracing"]["enabled"]
+
+  template "/tmp/trace_table.sql" do
+    source "phoenix_trace-table.sql.erb"
+    mode "0755"
+    action :create
+  end
+
+  bash "create_phoenix_trace_table" do
+    code <<-EOH
+    HBASE_CONF_PATH=/etc/hadoop/conf:/etc/hbase/conf /usr/hdp/current/phoenix-client/bin/sqlline.py "#{node[:bcpc][:hadoop][:zookeeper][:servers].map{ |s| float_host(s[:hostname])}.join(",")}:#{node[:bcpc][:hadoop][:zookeeper][:port]}:/hbase" /tmp/trace_table.sql
```

Comment:
This is a security issue as there is a race above I can have fun creating `/tmp/trace_table.sql` and now you are executing my arbitrary SQL as HBase; wahoo! (Please use the Chef cache.)